charts/dns/values.yaml

```diff
@@ -1,7 +1,8 @@
 externalDNS:
-  repository: k8s.gcr.io/external-dns/external-dns
-  pullPolicy: Always
+  repository: registry.k8s.io
```

Comment:
```suggestion
  registry: registry.k8s.io
```

Reply:
Good catch. I also have to set up my IDE linter because this should be easy to lint... Fixed in [#24e5b71](https://github.com/txqueuelen/pdns-stateless/pull/8/commits/24e5b712363b9bdce5c1dedd93033f0cb430371f)